### PR TITLE
fix(oie): Empty custom i18n value fix

### PR DESCRIPTION
**Pull request by leemchale-okta**
## Description:

This PR fixes an issue where an L10N_ERROR translation error is shown when overriding an i18n key with an empty string.  

**Before:**

Screenshot 2025-07-31 at 3 35 59 PM

**After:**

Screenshot 2025-07-31 at 3 34 55 PM

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-986305](https://oktainc.atlassian.net/browse/OKTA-986305)

### CODE REVIEW

**Review by alexchen-okta — DISMISSED** [review]

File: src/v2/ion/i18nUtils.js [hunk 1]
```diff
@@ -393,7 +393,7 @@ const isCustomizedI18nKey = (i18nKey, settings) => {
   
   // Lower case the language codes when searching for i18n key so it is case insensitive
   return Object.entries(i18n).some(
-    ([customizedLangCode, customizedI18nKeys]) => customizedLangCode.toLowerCase() === widgetLanguageLowerCase && customizedI18nKeys[i18nKey] !== undefined
+    ([customizedLangCode, customizedI18nKeys]) => customizedLangCode.toLowerCase() === widgetLanguageLowerCase && !!customizedI18nKeys[i18nKey]
```
Comment (Luxu2-Okta) [inline src/v2/ion/i18nUtils.js:396]: can we confirm Gen3 doesn't have this issue? I did some quick search [here](https://github.com/okta/okta-signin-widget/blob/4a014535b359b958ba8c957259f2f807025a2d84/src/util/loc.ts#L88-L89), it seems empty `value` will cause the issue in Gen3

Comment (leemchale-okta) [reply src/v2/ion/i18nUtils.js:396]: yeah actually my mistake, I tested locally before and didn't see the issue in gen 3 for some reason, but I just double checked and actually do see the issue now in gen 3 too.  Will make the same change in gen 3.

Comment (Luxu2-Okta) [reply src/v2/ion/i18nUtils.js:396]: Let's also add some test for empty string , unit test in `src/v3/src/transformer/i18n/isCustomizedI18nKey.test.ts`?

**Review by Luxu2-Okta — APPROVED** [review]
great fix! let's target 7.34.1
